Performance/js/harness-visualization.js

```diff
@@ -102,25 +102,16 @@
 
   // Populates a multiplier cell with the ratio between the given two values
   // and sets its background color depending on the magnitude.
-  function populateMultiplierCell(cell, language, swiftValue, otherValue) {
-    if (language != 'Swift') {
-      var multiplier = swiftValue / otherValue;
-
-      if (multiplier < 1) {
-        cell.text('(<1x)');
-      } else {
-        cell.text('(' + multiplier.toFixed(0) + 'x)');
-      }
-
-      if (multiplier < 3) {
-        cssClass = 'bg-success';
+    function decorateMultiplierCell(cell, multiplier) {
+      if (multiplier == 1) {
+          cell.addClass('bg-success');
+      } else if (multiplier < 3) {
+//        cssClass = 'bg-success';
```

Comment:
I've adjusted the comments to clarify that this case is deliberately doing nothing.  (This is responsible for the white cells above, which have multipliers >1 but <3.

I've also updated the comments and fixed the indentation for this function.